plugins/inventory/vmware_vm_inventory.py

```diff
@@ -11,20 +11,21 @@
     name: vmware_vm_inventory
     plugin_type: inventory
     short_description: VMware Guest inventory source
+    version_added: "2.7"
     author:
       - Abhijeet Kasurde (@Akasurde)
     description:
         - Get virtual machines as inventory hosts from VMware environment.
         - Uses any file which ends with vmware.yml, vmware.yaml, vmware_vm_inventory.yml, or vmware_vm_inventory.yaml as a YAML configuration file.
-        - The inventory_hostname is always the 'Name' and UUID of the virtual machine. UUID is added as VMware allows virtual machines with the same name.
     extends_documentation_fragment:
       - inventory_cache
+      - constructed
     requirements:
       - "Python >= 2.7"
       - "PyVmomi"
       - "requests >= 2.3"
-      - "vSphere Automation SDK - For tag feature"
-      - "vCloud Suite SDK - For tag feature"
+      - "vSphere Automation SDK"
+      - "vCloud Suite SDK"
```

Comment:
Why is ` - For tag feature` removed here? I tested your branch, and I think I'm running without this dependency, so it's not becoming a new dependency for all forms of inventory, right?

Reply:
It got inherited from #45 (reverted by #85). I'll revert that change.